### DISCUSSION TITLE
Welcome to SWR Discussions!

### DISCUSSION THREAD

**Discussion by shuding** [vercel/swr#1684] [General]
Hey everyone!

This is a place to ask questions or share thoughts publicly with the SWR community.

💬 You can also join the [Next.js Discord](https://nextjs.org/discord) (there is a SWR channel).
🐞 If you found a bug, it is best to [create an issue](https://github.com/vercel/swr/issues) with proper description and reproduction.
❓ If you have a question, search the existing issues and discussions. Feel free to start a new discussion if there isn't.

⌨️ To contribute to the project, please read our [Contributing Guidelines](https://github.com/vercel/swr/blob/main/.github/CONTRIBUTING.md).
📝 To contribute to the documentation, please take a look at the [SWR Site repository](https://github.com/vercel/swr-site).

And finally, the [Code of Conduct](https://github.com/vercel/swr/blob/main/.github/CODE_OF_CONDUCT.md) still applies here. We want to make this a place that everyone feels welcome!

**Comment by qiaogaolong** [comment]
Please help me deal with some bugs or reply to them. Thank you.
